libs/sr-ui/src/Context/DatabaseContext.tsx

```diff
@@ -1,15 +1,26 @@
 import { DBLocalStorage, SandboxStorage } from '@genshin-optimizer/database'
 import { SroDatabase } from '@genshin-optimizer/sr-db'
 import type { ReactNode } from 'react'
-import { createContext, useCallback, useMemo, useState } from 'react'
+import {
+  createContext,
+  useCallback,
+  useContext,
+  useMemo,
+  useState,
+} from 'react'
 
 type DatabaseContextObj = {
   databases: SroDatabase[]
   setDatabase: (index: number, db: SroDatabase) => void
   database: SroDatabase
 }
+
 export const DatabaseContext = createContext({} as DatabaseContextObj)
 
+export function useDatabaseContext() {
```

Comment:
Is this better than 2 imports? I guess it doesnt really matter, but I think you should try to enforce one vs the other. If you want to use this hook exclusively, do not export `DatabaseContext` so only this hook can be imported and used.